packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/MutableRealmIntInternal.kt

```diff
@@ -83,3 +88,15 @@ internal class UnmanagedMutableRealmInt(
 
     override fun decrement(value: Number) = increment(-value.toLong())
 }
+
+internal object ManagedMutableRealmIntSerializer : KSerializer<MutableRealmInt> {
```

Comment:
Just food for though, but would it make more sense to gather all the serializers in a single file? Since, if we ever have to change them, we probably need to change all of them?